mk/config.mk

```diff
@@ -4,6 +4,10 @@ CONFIGS_REPO_URL ?= https://github.com/betaflight/config
 CONFIGS_SUBMODULE_DIR := src/config
 BASE_CONFIGS           = $(sort $(notdir $(patsubst %/,%,$(dir $(wildcard $(CONFIG_DIR)/configs/*/config.h)))))
 
+ifneq ($(words $(CONFIG_DIR)),1)
+$(error CONFIG_DIR/BETAFLIGHT_CONFIG path contains whitespace; unsupported by GNU make wildcard.)
```

Comment:
Eh .. all this just to satisfy new AI overlords :-(

Reply:
Yeah this one was a waste... and the quotes previously and now this making the quotes unnecessary...